lib/resources/example_json/r4_examples_condition.rb

```diff
@@ -403,6 +403,22 @@ module Resources
                  ' Medical</p><p><b>Clinical Status</b>: Active</p><p><b>Verification Status</b>: Confirmed</p>'\
                  '<p><b>Onset</b>: Jun 10, 2020  5:00 P.M. UTC</p></div>'
         },
+        'extension': [
```

Comment:
You should add extenions like this also

https://fhir.cerner.com/millennium/r4/base/individuals/patient/#extensions

Reply:
https://fhir-ehr.cerner.com/r4/StructureDefinition/communication-preference?_format=json

Reply:
since these structure definition changes will be released in our next release, I will address your comments in a new pr during that release.

Reply:
since these structure definition changes will be released in our next release, I will address your comments in a new pr during that release.